packages/11ty/_plugins/shortcodes/cite.js

```diff
@@ -61,25 +61,25 @@ module.exports = function(eleventyConfig, { page }) {
     if (pageNumber) buttonText += divider + pageNumber
 
     const button = popupStyle === 'icon'
-      ? oneLine`
+      ? stripIndents`
           ${buttonText}
           <button class="quire-citation__button material-icons md-18 material-control-point" aria-expanded="false">
             control_point
           </button>
-        `
-      : oneLine`
+      `
+      : stripIndents`
           <span class="quire-citation__button" role="button" tabindex="0" aria-expanded="false">
             ${buttonText}
           </span>
-        `
+      `
 
-    return oneLine`
+    return stripIndents`
       <cite class="quire-citation expandable">
-        ${button}
+        ${button.replace(/\n/g, '')}
```

Comment:
this replace statement seems redundant as `replace` called on the string literal.